### PR TITLE
Use dune

### PR DESCRIPTION
**Pull request by samoht**
/cc @emillon and @hannesm

### CODE REVIEW

File: lib/dune [hunk 1]
```diff
@@ -0,0 +1,7 @@
+(library
+  (name        conduit)
```
Comment (emillon) [inline lib/dune]: you can drop the `(name)` and bump the version to 1.1 (lang & opam). If that's your thing :)

File: mirage-conduit.opam [hunk 2]
```diff
@@ -8,13 +8,13 @@ tags:         "org:mirage"
 license:      "ISC"
 
 build: [
-  ["jbuilder" "subst" "-n" name] {pinned}
-  ["jbuilder" "build" "-p" name "-j" jobs]
+  ["dune" "subst" "-n" name] {pinned}
```
Comment (emillon) [inline mirage-conduit.opam]: I'm not sure `dune subst` supports `-n`. Anyway it's safe to remove since it's in `dune-project`.

**Comment by samoht** [comment]
Any idea why ounit can't install on debian-unstable?

**Comment by emillon** [comment]
curl failed, it looks just transient. I'd retry the build.

**Comment by samoht** [comment]
still no luck :-(

**Comment by samoht** [comment]
Merging it anyway, as it seems to be an infrastructure failure.

**Comment by edwintorok** [comment]
I think the curl failure of ounit on debian unstable is because the version of openssl there drops support for TLS1.0 and the forge doesn't support it. There is a https://forge-static.ocamlcore.org/ mirror that would work.

**Pull request merged**